packages/cli/templates/platform-app-tailwind/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "sample-app",
+    "private": true,
+    "author": "Frontify Developers <developers@frontify.com>",
+    "version": "0.0.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "dev": "vite",
```

Comment:
```suggestion
```
We don't use the term `dev` in the templates so far, and most likely we don't want the usage of `vite` directly